### ISSUE TITLE
[SF][A11y][ui5-table] Table missing accessible name for screen reader users when viewed through screen reader shortcuts

### ISSUE THREAD

**Issue by 2bno1** [SAP/ui5-webcomponents#4648]
### **Bug Description**
Per https://github.com/SAP/ui5-webcomponents/blob/master/packages/main/src/Table.hbs#L6. ui5-table does not support accessible name for screen reader

### **Expected Behavior**
ui5-table should support accessible name for screen reader
Providing UI5 reference below showing accessible name for a table
![table_UI5](https://user-images.githubusercontent.com/1751846/151324978-9a95b5a1-9de3-460e-af52-f39d2e3aa879.png)

- UI5 Web Components version: 1.0.2
- OS/Platform: macOS
- Browser: Chrome
- Affected component: ui5-table

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

**Comment by LidiyaGeorgieva** [comment]
Hello @SAP/ui5-webcomponents-topic-rl, I'm forwarding you one feature request regarding ui5-table.

**Comment by elenastoyanovaa** [comment]
Hello @2bno1 ,

There is an existing request for the same #3597 . We will expose an API for setting accessible name for the table element. As this is a duplicate request I will close this one and you could track the progress in the other issue.

Thank you for reporting!
